features/developer_portal/authentication/sso_token.feature

```diff
@@ -0,0 +1,33 @@
+Feature: SSO Token
+  In order to sign in to developer portal using SSO token
+  As a buyer
+
+  Background:
+    Given a provider "foo.3scale.localhost"
+    And provider "foo.3scale.localhost" has multiple applications enabled
+    And a buyer "bob" signed up to provider "foo.3scale.localhost"
+    And the current domain is foo.3scale.localhost
+
+  Scenario: Buyer can't login with an invalid token
+    Given another provider generated a sso token for another buyer
+    When the buyer authenticates by SSO Token
+    Then the page should contain "Invalid SSO Token"
+
+  Scenario: Buyer can't login with an expired token
+    Given the provider generated a sso token for the buyer
```

Comment:
Just an improvement suggestion - the step could potentially have an optional suffix "valid for N minutes", that would create a token with `expires_in == N` (if missing, fall back to default 10 minutes).

It would make it clearer in this scenario that if the token is valid for 10 minutes, and 15 pass - that it would not be valid anymore.
But it's OK if you think that's too much.

Reply:
That's fine, I'll do it 👍

Reply:
done